### PR TITLE
hisat2: init 2.1.0

### PR DESCRIPTION
**Pull request by jbedo**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/biology/hisat2/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{stdenv, fetchFromGitHub, unzip, which, python}:
+
+stdenv.mkDerivation rec {
+  name = "hisat2-${version}";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "infphilo";
```
Comment (mpickering) [inline pkgs/applications/science/biology/hisat2/default.nix]: Why do you not get the source from the official release tarball? 

ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads

**Comment by mpickering** [comment]
@GrahamcOfBorg build hisat2

**Comment by mpickering** [comment]
Please fix the darwin build or modify the platforms attribute.

**Comment by mpickering** [comment]
@GrahamcOfBorg build hisat2

**Comment by jbedo** [comment]
Sorry keep getting the platforms slightly wrong, seems it assumes sse so has to be x86. I've tried to restrict to x86 linux only, is this the cleanest way?

**Comment by mpickering** [comment]
You can see the available platforms here - https://github.com/NixOS/nixpkgs/blob/e4d9ce90611a90d7855dc234fb33e89217cd16ed/lib/systems/for-meta.nix#L9

**Comment by jbedo** [comment]
Thanks, doesn't look any cleaner than just listing the two platforms.

**Pull request merged**